api/python/quilt3/telemetry.py

```diff
@@ -15,19 +15,26 @@
 TELEMETRY_URL = "https://telemetry.quiltdata.cloud/Prod/metrics"
 TELEMETRY_USER_AGENT = "QuiltCli"
 TELEMETRY_CLIENT_TYPE = "quilt3-python-client"
-TELEMETRY_SCHEMA_VERSION = "pyclient-usage-metrics-v1"
+TELEMETRY_SCHEMA_VERSION = "pyclient-usage-metrics-v2"
 
 DISABLE_USAGE_METRICS_ENVVAR = "QUILT_DISABLE_USAGE_METRICS"
 MAX_CLEANUP_WAIT_SECS = 5
 
 
 class ApiTelemetry:
     session = None
-    session_id = str(uuid.uuid4())
```

Comment:
Just a note, this will have implications for how we unique sessions. Need to note this in the analytics backend. I will show you.

Reply:
Per Slack this is still a class variable. Doesn't it still need to be declared?

Reply:
You can't "declare" variable in Python, the deleted line was assignment that is moved to `reset_session_id()`.

Reply:
IMHO as is this class doesn't add much value and it might worth to move all its stuff to the module level.

Reply:
I guess it's ok but it seems very weird for initialization to depend on a function call when another LOC when eg report_api_use calls it

Reply:
ie much cleaner to init in original location